payments/schema/mutations.py

```diff
@@ -859,6 +863,43 @@ def mutate_and_get_payload(cls, root, info, application_id, new_berth_id, **inpu
         return CreateBerthSwitchOfferMutation(berth_switch_offer=offer)
 
 
+class AcceptBerthSwitchOfferMutation(graphene.ClientIDMutation):
+    class Input:
+        offer_id = graphene.ID(required=True)
+        is_accepted = graphene.Boolean(required=True)
+
+    offer = graphene.Field(BerthSwitchOfferNode)
```

Comment:
Not 100% about this 🤔 since the mutation is being executed by the customer UI, it wouldn't expect any response. And if it did, I think it should also return the new lease 🤔 🤔 🤔 🤔

Reply:
Ok, since now we'll have "both" APIs living under tha same roof, let's keep the offer and also add the new lease, and then let the UI decide what information they need back.

Reply:
Yep, sounds good to me.

Reply:
New lease added to response.

Reply:
I checked Lease and Offer, to me it seems they don't have any fields that the customer isn't allowed to see (it's their lease and if the customer made a GDPR request, the city would need to provide the data anyway I think).